### DISCUSSION TITLE
Old commits transfer

### DISCUSSION THREAD

**Discussion by sovetski** [community/community#56549] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think we should be able to transfer our old commits to a specific account, especially if we transform an account to an organization.

In this page: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization

We have this information

> Any commits made with the converted personal account will no longer be linked to that account. The commits themselves will remain intact.

Why? It is important to be able to transfer all organization previous personal commits to the new GitHub organization owner accounts or other. In my case, I want this because I want to display the commits in the graph on my personal account.

To summarize, here is what will be perfect as a feature

- If you transform a personal account into an organization, either the commits are automatically transferred to the account of the organization owner, or you choose which account to transfer to
- A tool that allows changing the author of commits from any repository. To add more security, the person who made the commits can receive an email to confirm the transfer etc.

It remains on my opinion/suggestion, it can surely improve, but it will be really great to have these features

**Comment by airtower-luna** [comment]
GitHub assigns commits to users based on the *author email* recorded in the commit. You'll have to add the author address that you used for those commits (and that I assume is/was on the account you want to turn into an organization) to your personal account. Then they should show up as yours.

**Reply by sovetski** [reply]
Thank you for your answer ! Indeed, it can be an alternative solution, but let's imagine this scenario:

A is my current account, B is the personal account I want to turn into an organization.

If in account B, for example I used dev@myorg.com as author of the commits (before transformation into organization account), if I add this email in my account A, later it can cause problems, because an organization can grow and belong to someone else.

So if the domain name myorg.com needs to be transferred to someone else, in this case I would find myself in a complicated situation
